Diagnostic/HandlerManifest.json

```diff
@@ -1,15 +1,16 @@
 [{
-  "name": "LinuxAzureDiagnostic",
-  "version": 1.0,
-  "handlerManifest": {
-   "installCommand": "diagnostic.py -install",
-    "uninstallCommand": "diagnostic.py -uninstall",
-    "updateCommand": "diagnostic.py -update",
-    "enableCommand": "diagnostic.py -enable",
-    "disableCommand": "diagnostic.py -disable",
-    "rebootAfterInstall": false,
-    "reportHeartbeat": false,
+    "namespace": "Microsoft.OSTCExtensions",
+    "name": "LinuxDiagnostic",
+    "version": "2.3.9001",
```

Comment:
I've heard from @ahmetalpbalkan that this version is not the extension version, but the handler version, so it should be all 1.0. Ahmet should confirm.